apis/v1alpha1/generator.yaml

```diff
@@ -2,10 +2,28 @@ ignore:
   field_paths: null
   operations: null
   resource_names:
-  - Trail
   - EventDataStore
   shape_names: null
 model_name: cloudtrail
 operations: {}
 prefix_config: {}
-resources: {}
+resources:
+  Trail:
+    fields:
+      Name:
+        is_immutable: true
+        is_required: true
+      S3BucketName:
+        is_required: true
```

Comment:
Both of these are required in the `CreateTrailInput`. Were they not being set as required by default?

Reply:
Just a reflex that i had from early implementations.. removing them